private/buf/cmd/buf/command/push/push.go

```diff
@@ -346,6 +346,37 @@ func createUploadModulesIfNotExist(
 	return nil
 }
 
+func validateFlags(flags *flags) error {
+	if err := validateCreateFlags(flags); err != nil {
+		return err
+	}
+	if err := validateLabelFlags(flags); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateLabelFlags(flags *flags) error {
+	for _, label := range flags.Labels {
+		if label == "" {
+			return appcmd.NewInvalidArgumentErrorf(
+				"--%s requires a non-empty string.",
+				labelFlagName,
+			)
+		}
+	}
+	for _, tag := range flags.Tags {
+		if tag == "" {
+			return appcmd.NewInvalidArgumentErrorf(
+				"-%s|--%s requires a non-empty string.",
```

Comment:
All other usages of `appcmd.NewInvalidArgumentErrorf` (save one) in the codebase do not display short names, and do not use punctuation - update to `--%s requires a non-empty string`

Reply:
Done!

Reply:
Did you want me to remove the trailing period on the errors returned by `validateCreateFlags`?

Reply:
Yea for sure, that was a miss on my part

Reply:
Done!